lib/forgery.ex

```diff
@@ -132,12 +149,12 @@ defmodule Forgery do
 
   See `Kernel.struct!/2` for more information.
 
-      iex> create_struct(%{id: 1, name: "john", password: "123456"}, User)
-      %User{id: 1, password: "123456", name: "john"}
+      iex> create_struct(%{id: 1, name: "John", password: "123456"}, User)
+      %User{id: 1, password: "123456", name: "John"}
 
   """
 
-  @spec create_struct(fields :: Enumerable.t(), module :: atom()) :: struct()
+  @spec create_struct(fields :: Enumerable.t(), module()) :: struct()
```

Comment:
```suggestion
  @spec create_struct(fields :: Enumerable.t(), module()) :: %module{}
```

??

Reply:
It actually can be `@spec create_struct(fields :: Enumerable.t(), struct()) :: struct()`

Reply:
And the `%module{}` is in fact invalid.